### PR TITLE
[estuary] sync

### PR DESCRIPTION
**Pull request by phil65**
Adds some missing timeshift functionality (tested by @ksooo )
@ronie

### CODE REVIEW

**Comment by ksooo** [comment]
+1 from my side

File: addons/skin.estuary/xml/DialogFullScreenInfo.xml [hunk 1]
```diff
@@ -18,69 +18,72 @@
...
+						<width>1000</width>
 						<height>100</height>
-						<textoffsetx>80</textoffsetx>
-						<label>$LOCALIZE[19031]</label>
-						<visible>VideoPlayer.Content(livetv)</visible>
+						<align>right</align>
+						<orientation>horizontal</orientation>
+						<itemgap>-10</itemgap>
+						<onup>Close</onup>
+						<ondown>Close</ondown>
```
Comment (ronie) [inline addons/skin.estuary/xml/DialogFullScreenInfo.xml]: Dialog.Close(fullscreeninfo) would be nicer imo

(fwiw, "close" is deprecated: https://github.com/xbmc/xbmc/blob/master/xbmc/input/ButtonTranslator.cpp#L116)

**Comment by phil65** [comment]
thx, updated.

**Comment by ksooo** [comment]
@ronie good to go?

**Comment by ronie** [comment]
sure sure

**Pull request merged**